src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/Models/Database.cs

```diff
@@ -0,0 +1,305 @@
+        }
+
+        /// <summary>
+        /// Gets or sets the collation of the Azure SQL Database.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "properties.collation")]
+        public string Collation { get; set; }
+
+        /// <summary>
+        /// Gets the creation date of the Azure SQL Database (ISO8601 format).
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "properties.creationDate")]
+        public System.DateTime? CreationDate { get; private set; }
+
+        /// <summary>
+        /// Gets the containment state of the Azure SQL Database.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "properties.containmentState")]
+        public long? ContainmentState { get; private set; }
+
+        /// <summary>
+        /// Gets the current Service Level Objective Id of the Azure SQL
+        /// Database. This is the Id of the Service Level Objective that is
+        /// currently active.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "properties.currentServiceObjectiveId")]
+        public System.Guid? CurrentServiceObjectiveId { get; private set; }
+
+        /// <summary>
+        /// Gets the Id of the Azure SQL Database.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "properties.databaseId")]
+        public string DatabaseId { get; private set; }
+
+        /// <summary>
+        /// Gets the recovery period start date of the Azure SQL Database.
+        /// This records the start date and time when recovery is available
+        /// for this Azure SQL Database (ISO8601 format).
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "properties.earliestRestoreDate")]
+        public System.DateTime? EarliestRestoreDate { get; private set; }
+
+        /// <summary>
+        /// Gets or sets specifies the type of database to create. Possible
+        /// values include: 'Copy', 'Default', 'NonReadableSecondary',
+        /// 'OnlineSecondary', 'PointInTimeRestore', 'Recovery', 'Restore'
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "properties.createMode")]
+        public string CreateMode { get; set; }
+
+        /// <summary>
+        /// Gets or sets conditional.  Specifies the resource Id of the source
+        /// database.  If createMode is not set to Default, then this value
+        /// must be specified.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "properties.sourceDatabaseId")]
+        public string SourceDatabaseId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the edition of the Azure SQL Database.  The
+        /// DatabaseEditions enumeration contains all the valid editions.
+        /// Possible values include: 'Web', 'Business', 'Basic', 'Standard',
+        /// 'Premium', 'Free', 'Stretch', 'DataWarehouse'
+        /// </summary>
```

Comment:
This is generated from the swagger, so I will address this in a separate PR for the specs.